### PR TITLE
Feat: WDF Overview

### PR DESCRIPTION
**Pull request by joannafawl**
#### Work done
- Create new WDF overview page for happy path
- Use `reportService` to get WDF dates

#### Tests
Does this PR include tests for the changes introduced?
- [x] Yes
- [ ] No, I found it difficult to test
- [ ] No, they are not required for this change

### CODE REVIEW

File: src/app/features/wdf/wdf-overview/wdf-overview.component.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+    this.subscriptions.add(
+      this.reportService.getWDFReport(this.workplace.uid).subscribe((report) => {
+        this.report = report;
+        this.wdfStartDate = moment(this.report.effectiveFrom).format('D MMMM YYYY');
+        this.wdfEndDate = moment(this.report.effectiveFrom).add(1, 'years').format('D MMMM YYYY');
+        this.overallEligibilityDate = moment(this.report.wdf.overallWdfEligibility).format('D MMMM YYYY');
+      }),
+    );
+  }
```
Comment (chadtomkiss-madetech) [inline src/app/features/wdf/wdf-overview/wdf-overview.component.ts:41]: Missing an `ngOnDestroy` to clear the subscriptions :eyes:

File: src/app/features/wdf/wdf-overview/wdf-overview.component.html [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+          src="/assets/images/tick-icon.svg"
+          alt=""
+        />Your data has met the WDF {{ wdfStartDate | date: 'yyyy' }} to {{ wdfEndDate | date: 'yyyy' }} requirements
+      </h2>
+    </span>
+    <p>
+      Your data met the requirements on {{ overallEligibilityDate }} and will continue to meet them until the next fund
+      is available on {{ wdfEndDate }}.
+    </p>
+    <p><a [routerLink]="['/reports']">Keep your WDF data up to date</a>, it will save you time next year.</p>
```
Comment (chadtomkiss-madetech) [inline src/app/features/wdf/wdf-overview/wdf-overview.component.html:18]: Should this go to `/reports`? I would maybe have an empty link until we have the new page to go to

File: src/app/features/wdf/wdf-overview/wdf-overview.component.spec.ts [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  };
+
+  it('should render a WdfOverviewComponent', async () => {
+    const { component } = await setup();
+    expect(component).toBeTruthy();
+  });
+
+  it('should display the correct timeframe for meeting WDF requirements', async () => {
+    const { getByText } = await setup();
+    const timeframeSentence = 'Your data has met the WDF 2021 to 2022 requirements';
```
Comment (chadtomkiss-madetech) [inline src/app/features/wdf/wdf-overview/wdf-overview.component.spec.ts:37]: Will these tests fail next year? We might need to fake the current date/time here

**Review by chadtomkiss-madetech — CHANGES_REQUESTED** [review]

File: src/app/features/wdf/wdf-overview/wdf-overview.component.spec.ts (hunk 3 above)
Comment (chadtomkiss-madetech) [reply src/app/features/wdf/wdf-overview/wdf-overview.component.spec.ts:37]: Ignore me, the dates are hardcoded in the mock service 🙈

File: src/app/features/wdf/wdf-overview/wdf-overview.component.html (hunk 2 above)
Comment (duncanc19) [reply src/app/features/wdf/wdf-overview/wdf-overview.component.html:18]: Yeah, we had thought that but an empty link redirects to the home page, is that better? I guess it's clearer that it needs to be changed?

Comment (chadtomkiss-madetech) [reply src/app/features/wdf/wdf-overview/wdf-overview.component.html]: I think so.. it looks like it's linked up properly if you're not aware new screens are coming up

**Review by chadtomkiss-madetech — APPROVED** [review]

**Pull request merged**